setup.py

```diff
@@ -35,7 +35,7 @@
     ),
     python_requires='>=2.7',
     install_requires=[
-        'pyyaml',
+        'yamlenv',
```

Comment:
Perhaps since the library still references `import yaml`, the requirement on pyyaml should be explicit? 

```suggestion
        'pyyaml',
        'yamlenv',
```